art/attacks/evasion/__init__.py

```diff
@@ -48,3 +48,4 @@
 from art.attacks.evasion.virtual_adversarial import VirtualAdversarialMethod
 from art.attacks.evasion.wasserstein import Wasserstein
 from art.attacks.evasion.zoo import ZooAttack
+from art.attacks.evasion.pe_malware_attack import MalwareGD
```

Comment:
Could you please move the new entry to the line after NewtonFool? The list aims to be sorted by the module name.